src/DynamoCoreWpf/UI/Prompts/DynamoMessageBox.xaml.cs

```diff
@@ -123,6 +129,24 @@ public DynamoMessageBox()
             return dynamoMessageBox.CustomDialogResult;
         }
 
+        public static MessageBoxResult Show(string messageBoxText, string caption, MessageBoxButton button,
```

Comment:
since it's 3.0 what about getting rid of the other constructor and making these  new parameters optional?